lib/ansible/modules/windows/win_shortcut.ps1

```diff
@@ -26,7 +26,7 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
 
-$src = Get-AnsibleParam -obj $params -name "src"
```

Comment:
Because it could be a url. It's unfortunate we didn't have 2 options `src` and `url` in this case.
But like many other modules the `src` accepts different input.

Maybe we could make a new standard, phase out the old model and wait until v2.8 to clean it up ?